runtime/src/configs/mod.rs

```diff
@@ -1,3 +1,28 @@
+// This is free and unencumbered software released into the public domain.
```

Comment:
Again... 😂

Reply:
Yeah that's the issue when creating PRs over unmerged branches 😬

Reply:
Oh, it's a different file, nvm I'm just dumb